### ISSUE TITLE
[Usable] Reading 'Group' between the text boxes while navigating in scan mode.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1349]
**Actual Result:**
Narrator is reading 'Group', on Gap between the text boxes while navigating in scan mode.

**Note** - Same issue repro at - 
1. URL -> Help -> Show Restaurant information using  Adaptive card Button ->New Content -> Call Restaurant - Narrator is reading 'Group', Gap between the text boxes while navigating in scan mode.
Attachment **"Usable_Reading Group".**

**Expected Result:**
Narrator should not read 'Group', on Gap between the text boxes while navigating in scan mode.

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/ ) in Edge browser.
2. Notice 'Upload File' and 'Send' button icons present in bottom of screen.
3. Type 'Help' in the input control at the bottom of the screen and hit enter (or click on send button).
4. Content will upload, Navigate to any "Show an Adaptive cards with all types of inputs " button by using Tab key and press Enter on it.
5. New data will upload.
6. Open Narrator with scan mode (Caps+Space)
7. Navigate to text boxes by using Up/Down arrow keys.
8. Observe that narrator is reading 'Group' between the text boxes while navigating in scan mode.

**User Impact:**
If narrator reads group between the text boxes ,User who is dependent on screen reader, will get confuse about the purpose of the controls.

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17134.345
**OS Version:** 1803
**Browser:** Edge
**Tool:** Narrator

**Attachment:**
1. [Usable_Reding group between text boxes.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2579388/Usable_Reading.group.between.text.boxes.zip)
2. [Usable_Reading Group.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2584539/Usable_Reading.Group.zip)

**Comment by compulim** [comment]
The whole card (including the input text boxes) are generated by Adaptive Card. This bug should belongs to Adaptive Card project.

**Comment by corinagum** [comment]
Adaptive Cards issue can be found [here](https://github.com/Microsoft/AdaptiveCards/issues/2239).

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yUsable

**Comment by pgopinadhu** [comment]
#A11yUsable;#A11ySev4;
